ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/port-channel-interfaces.j2

```diff
@@ -76,21 +76,37 @@ port_channel_interfaces:
     mtu: {{ adapter_settings.mtu }}
 {%                         endif %}
 {%                         if adapter_settings.switches | unique | list | length > 1 %}
-{%                             if adapter_settings.port_channel.short_esi is arista.avd.defined %}
-{%                                 if adapter_settings.port_channel.short_esi | lower == 'auto' %}
+{%                             if adapter_settings.port_channel.short_esi is arista.avd.defined and adapter_settings.ethernet_segment.short_esi is not arista.avd.defined %}
+{%                                 do adapter_settings.update({ 'ethernet_segment': { 'short_esi': adapter_settings.port_channel.short_esi }  }) %}
+{%                             endif %}
+{%                             if adapter_settings.ethernet_segment.short_esi is arista.avd.defined %}
+{%                                 if adapter_settings.ethernet_segment.short_esi | lower == 'auto' %}
 {%                                     set esi_hash_switches = adapter_settings.switches[:2] | join %}
 {%                                     set esi_hash_switch_ports = adapter_settings.switch_ports[:2] | join %}
 {%                                     set esi_hash_endpoint_ports = adapter_settings.endpoint_ports[:2] | join %}
 {%                                     set esi_hash = (esi_hash_switches ~ esi_hash_switch_ports ~ esi_hash_endpoint_ports ~ channel_group_id) | hash('sha256') %}
 {%                                     set short_esi_hash = (esi_hash | regex_replace("([0-9a-f]{4})","\\1:"))[:14] %}
-{%                                     do adapter_settings.port_channel.update({ 'short_esi': short_esi_hash }) %}
+{%                                     do adapter_settings.ethernet_segment.update({ 'short_esi': short_esi_hash }) %}
 {%                                 endif %}
-{%                                 if adapter_settings.port_channel.short_esi.split(':') | length == 3 %}
+{%                                 if adapter_settings.ethernet_segment.short_esi.split(':') | length == 3 %}
     evpn_ethernet_segment:
-      identifier: {{ adapter_settings.port_channel.short_esi | arista.avd.generate_esi(evpn_short_esi_prefix) }}
-      route_target: {{ adapter_settings.port_channel.short_esi | arista.avd.generate_route_target }}
+      identifier: {{ adapter_settings.ethernet_segment.short_esi | arista.avd.generate_esi(evpn_short_esi_prefix) }}
+{%                                     if adapter_settings.ethernet_segment.redundancy is arista.avd.defined %}
+      redundancy: {{ adapter_settings.ethernet_segment.redundancy }}
+{%                                     endif %}
+{%                                     if adapter_settings.ethernet_segment.designated_forwarder_preferences is arista.avd.defined %}
+      designated_forwarder_election:
+        algorithm: preference
```

Comment:
So from what I read it may be on purpose but I is it not possible to configure `modulus` here? (or do we rely on the fact that by default it is modulus maybe?)

Reply:
Yes, I had a mental note to add that into port-channel interfaces; wasn't a big deal as the EOS default is modulus, but it's nice to have things consistent between the two.